### PR TITLE
Update apply page for Spring 2020

### PR DESCRIPTION
**Pull request by davish**

### CODE REVIEW

File: pages/apply.js [hunk 1]
```diff
@@ -73,10 +116,18 @@ const Apply = () => (
               evaluate through the technical challenge phase of the application
               process.
               <br />
-              <br /> Come out to our info sessions <b>Tuesday 9/3</b> and{' '}
-              <b>Thursday 9/5</b> from <b>6-7pm in JMHH 340</b> to learn more
+              <br /> Come out to our info sessions to learn more
               and meet our members! And feel free to reach out to
               contact@pennlabs.org anytime.
```
Comment (ArmaanT) [inline pages/apply.js]: Can we make this a clickable link?

File: pages/apply.js [hunk 2]
```diff
@@ -1,24 +1,67 @@
 import Layout from '../components/Layout'
 import Link from 'next/link'
 
+const info = {
+  isOpen: true,
+  applicationUrl: 'https://airtable.com/shrvvvrVR4E2VZDUo',
+  season: 'Spring',
+  year: '2020',
+  sessions: [
```
Comment (ArmaanT) [inline pages/apply.js:9]: Let's wait until we get confirmation on the rooms for the info session and update this before deploying

**Review by ArmaanT — CHANGES_REQUESTED** [review]
One minor change. See my comment on waiting to deploy though

File: pages/apply.js (hunk 2 above)
Comment (davish) [reply pages/apply.js:9]: for sure. was planning on doing that

**Review by ArmaanT — APPROVED** [review]

**Pull request merged**